curator-recipes/src/test/java/org/apache/curator/framework/recipes/leader/ChaosMonkeyCnxnFactory.java

```diff
@@ -59,13 +59,6 @@ public void startup(ZooKeeperServer zks) throws IOException, InterruptedExceptio
         super.startup(new ChaosMonkeyZookeeperServer(zks));
     }
 
-    /**
```

Comment:
Not sure? I don't see this change in my revision history? Last update was 6 months ago in the upgrade to ZK 3.5.0-alpha by Ioannis?